packages/manager/apps/public-cloud/src/assets/theme/_navbar.less

```diff
@@ -8,7 +8,7 @@
 .oui-navbar {
   background-color: #fff;
   box-shadow: 0 2pt 4pt rgba(17, 63, 109, 0.15);
-  z-index: 2;
+  z-index: 3;
```

Comment:
I think you'll need to increase the `sidebar` z-index as it is not related the navbar anymore 
https://github.com/ovh-ux/manager/blob/91336be8f23e6df3468a9dfc420b4dce1ca97ff7/packages/manager/apps/public-cloud/src/assets/theme/_sidebar.less#L16

Reply:
Sorry. My mistake. Its navbar and not sidebar. Navbar is the one that has links to Manager Web, dedicated etc